inltk/tokenizer.py

```diff
@@ -55,7 +55,7 @@ def __init__(self, lang: str):
         LanguageTokenizer.__init__(self, lang)
 
 
-class MalyalamTokenizer(LanguageTokenizer):
+class MalayalamTokenizer(LanguageTokenizer):
```

Comment:
@adamshamsudeen did you try and run inltk after making these changes? This won't work whenever malayalam language is being used because the malayalam model being loaded will look for class MalyalamTokenizer as it has reference stored by that name. You'll have to also make changes to the class name [here](https://github.com/goru001/nlp-for-malyalam/tree/master/language-model) and export the model, store it on dropbox and update the link to model [here](https://github.com/goru001/inltk/blob/master/inltk/config.py#L29)